src/primitives/arc/linear_equation.rs

```diff
@@ -20,10 +20,17 @@ pub struct LinearEquation {
 impl LinearEquation {
     /// Create a new linear equation based on one point and one angle
     pub fn from_point_angle(point: Point, angle: Angle) -> Self {
-        let (a, b) = match angle.tan() {
-            None => (Real::from(1.0), Real::from(0.0)),
-            Some(a) => (-a, Real::from(-1.0)),
+        // FIXE: angle.tan() for 180.0 degrees isn't exactly 0 which causes problems when drawing
+        //       a single quadrant. Is there a better solution to fix this?
```

Comment:
```suggestion
        // FIXME: angle.tan() for 180.0 degrees isn't exactly 0 which causes problems when drawing
        //        a single quadrant. Is there a better solution to fix this?
```

A stupid nit, but makes for more accurate grepping